packages/components/checkbox/src/checkbox.vue

```diff
@@ -65,18 +69,21 @@
 
 <script lang="ts" setup>
 import { useSlots } from 'vue'
-import { useNamespace } from '@element-plus/hooks'
+import { useAttrs, useNamespace } from '@element-plus/hooks'
 import { checkboxEmits, checkboxProps } from './checkbox'
 import { useCheckbox } from './composables'
 
 defineOptions({
   name: 'ElCheckbox',
+  inheritAttrs: false,
```

Comment:
I remember someone reporting that this would cause attributes other than class and style to be lost, will `@click.stop` be useful?

Reply:
Yeah, I noticed this issue https://github.com/element-plus/element-plus/issues/8641. Maybe we need to discuss where the custom attributes should be placed? 🤔
And `@click.stop` doesn't work on click of label.